packages/tests/e2e/robot/tests/api/resources/packager/valid/docker_rest_create.json

```diff
@@ -0,0 +1,154 @@
+{
+  "kind": "PackagingIntegration",
+  "id": "docker-rest-api-testing",
+  "spec": {
+    "entrypoint": "created",
+    "defaultImage": "created",
+    "privileged": true,
+    "schema": {
+      "targets": [
+        {
+          "name": "docker-push",
+          "connectionTypes": [
+            "docker",
+            "ecr"
+          ],
+          "default": "docker-ci",
+          "required": true
+        },
+        {
+          "name": "docker-pull",
+          "connectionTypes": [
+            "docker",
+            "ecr"
+          ],
+          "default": "odahuflow-docker-repository",
+          "required": false
+        }
+      ],
+      "arguments": {
+        "properties": [
+          {
+            "name": "dockerfileBaseImage",
+            "parameters": [
+              {
+                "name": "description",
+                "value": "Base image for Dockerfile."
+              },
+              {
+                "name": "type",
+                "value": "string"
+              },
+              {
+                "name": "default",
+                "value": "gcr.io/or2-msq-epmd-legn-t1iylu/odahu/odahu-flow-docker-packager-base:1.2.0-b1592953604635"
```

Comment:
Is it safe to use dev build number? Maybe we can pass this as argument or use some permanent tag?